projects/storefrontlib/src/lib/product/components/product-details/product-images/docs/custom-styles.md

```diff
@@ -0,0 +1,93 @@
+# Custom Styles
+
+## Vertical thumb panel
+
+The folowing snippets demonstrates the so-called amazon layout of thumbs. It's a small customisation that renders the thumbnails on the left hand side of the main image.
+
+```
+cx-product-images {
+  --cx-flex-direction: row-reverse;
+  justify-content: flex-end;
+
+  .thumbs {
+    flex-direction: column;
+  }
+}
+```
+
+**Demo**
+
+![amazon-layout](amazon-layout.gif)
+
+## Staggered thumbnails
+
+The PDP images can take quite some space. This example demonstrates staggered images, placed on top of the main image. Whenever the user navigates to the thumbnails, the stagered images animate to use the original space.
```

Comment:
Not sure about the staggered name, maybe "stacked". If you keep the staggered then there is a typo in the second one in the description.

Reply:
Stacked, much better!